src/WebJobs.Extensions.DurableTask/OutOfProcMiddleware.cs

```diff
@@ -43,6 +43,10 @@ public OutOfProcMiddleware(DurableTaskExtension extension)
         /// <exception cref="SessionAbortedException">Thrown if there is a recoverable error in the Functions runtime that's expected to be handled gracefully.</exception>
         public async Task CallOrchestratorAsync(DispatchMiddlewareContext dispatchContext, Func<Task> next)
         {
+#if FUNCTIONS_V1
+            await Task.Yield(); // Used to supress CS1998: This async method lacks "await" operators (by design)
```

Comment:
Is there a reason why we wouldn't just suppress this warning?

Reply:
It requires suppressing it at the method declaration level, which means you can only apply the suppression to the whole method or not at all. I only wanted to suppress it for `FUNCTIONS_V1`, so I suppressed in in this way.